.travis.yml

```diff
@@ -1 +1,16 @@
 language: node_js
+cache:
+  directories:
+    - node_modules
+notifications:
+  email: false
+node_js:
+  - '7'
```

Comment:
No, semantic-release is smart enough to wait for all builds to finish, only publishing from the first build.